intranet/apps/dataimport/management/commands/import_users.py

```diff
@@ -0,0 +1,124 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth import get_user_model
+from django.db.models import Q
+import csv
+import json
+import sys
+
+class Command(BaseCommand):
+    help = "Import Fake People"
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('data_fname')
+       
+
+    def clean(self, name):
+        return name.replace(" ", "").replace("-","").replace("_","").replace("'", "")
+    
+    def handle(self, *args, **kwargs):
+        with open(kwargs["data_fname"]) as f:
+            data = json.load(f)
+
+        
+        counselor_data = data["counselors"]
+        
+
+        for counselor in range(len(counselor_data)):
+            gender = counselor_data[counselor]["Gender"].strip()
+            grade = counselor_data[counselor]["Grade"]
+            first_name = counselor_data[counselor]["First Name"].strip()
+            last_name = counselor_data[counselor]["Last Name"].strip()
+            full_name = counselor_data[counselor]["Full Name"].strip()
+            username = counselor_data[counselor]["User Name"].strip()
+            if gender == 'M':
+                gender = 'Male'
+            else:
+                gender = 'Female'
+
+            u = get_user_model().objects.get_or_create(username=username,
+                defaults = {
+                    'last_name':last_name,
```

Comment:
I would use double quotes. In addition, our linters will complain about a missing space between the key and value.